requirements_test_all.txt

```diff
@@ -104,7 +104,7 @@ libpurecoollink==0.4.2
 libsoundtouch==0.7.2
 
 # homeassistant.components.device_tracker.google_maps
-locationsharinglib==3.0.2
+locationsharinglib==3.0.3
```

Comment:
There are no tests for the google_maps platform.

Reply:
I see...was added with #16969 and is a mistake.